README.md

```diff
@@ -360,6 +367,72 @@ The [`FairplayConfig`](https://github.com/bitmovin/bitmovin-player-react-native/
 
 Also, don't forget to check out the [example](https://github.com/bitmovin/bitmovin-player-react-native/tree/development/example) app for a complete iOS/Android [DRM example](https://github.com/bitmovin/bitmovin-player-react-native/blob/development/example/src/screens/BasicDRMPlayback.tsx).
 
+### Adding external subtitle tracks
+
+Usually, subtitle tracks are provided in the manifest of your content (see [Enconding Manifests API](https://bitmovin.com/docs/encoding/api-reference/sections/manifests) for more information). And if they are provided this way, the player already recognizes them and show them in the subtitles selection menu without any further configuration.
+
+Otherwise, it's also possible to add external tracks via the subtitle API:
+
+```typescript
+import { Platform } from 'react-native';
+import {
+  SourceConfig,
+  SourceType,
+  SubtitleFormat,
+} from 'bitmovin-player-react-native';
+
+// Source config with an external subtitle track.
+const config: SourceConfig = {
+  url:
+    Platform.OS === 'ios'
+      ? 'https://bitmovin-a.akamaihd.net/content/sintel/hls/playlist.m3u8'
+      : 'https://bitmovin-a.akamaihd.net/content/sintel/sintel.mpd',
+  type: Platform.OS === 'ios' ? SourceType.HLS : SourceType.DASH,
+  poster: 'https://bitmovin-a.akamaihd.net/content/sintel/poster.png',
+  // External subtitle tracks list to be added to this source.
+  addSubtitleTracks: [
```

Comment:
On android this is already a property and it is called `subtitleTracks`.
_In case we would like to have a different naming on RN, I would rather go for `sideLoadedSubtitleTracks` instead of `externalSubtitleTracks`._

Reply:
I see, no problem. I think it'd be better to rename it to `subtitleTracks` then. So Android and iOS get to use the same naming pattern. What do you think?

Reply:
Done f0b320c3a06dd85fd5386ba44ce5ad0b02f0331d